hs_composite_resource/serialization.py

```diff
@@ -0,0 +1,6 @@
+
+from hs_core.serialization import GenericResourceMeta
+
+
+class CompositeResourceMeta(GenericResourceMeta):
```

Comment:
Is it a bad idea to rename GenericResourceMeta ->  CompositeResourceMeta throughout the codebase instead? More or less confusing?

Reply:
ah good point, thanks for contemplating. Let's stick with what you have